yt/odin/checks/bin/unaware_nodes/__main__.py

```diff
@@ -1,25 +1,19 @@
 from yt_odin_checks.lib.check_runner import main
 
-DEFAULT_ENABLED_NODE_FLAVORS = [
-    "data",
-    "exec",
-    "tablet",
-    "chaos"
-]
 
-
-def find_unaware_nodes(yt_client, check_node_flavors):
+def find_unaware_nodes(yt_client, node_flavors_to_check):
     result = []
     for row in yt_client.list("//sys/cluster_nodes", attributes=["rack", "flavors"], read_from="cache"):
-        flavors = row.attributes.get("flavors", [])
+        node_flavors = row.attributes.get("flavors", [])
 
-        check_node_rack = False
-        for node_flavor in flavors:
-            if node_flavor in check_node_flavors:
-                check_node_rack = True
-                break
-        if not check_node_rack and len(flavors) > 0:
-            continue
+        if node_flavors_to_check:
+            check_node_rack = False
+            for flavor in node_flavors:
+                if flavor in node_flavors_to_check:
+                    check_node_rack = True
+                    break
+            if not check_node_rack and node_flavors:
```

Comment:
Why do we check non-emptyness of node_flavors here? Each node at least has a "cluster" flavor.

Reply:
I'd looked into a couple of clusters available and didn't see such a flavor. I DID see it in a `.h` file but it has been marked as a deprecated one.